assets/prefabs/blocks/effectors/EffectorOne.prefab

```diff
@@ -1,6 +1,6 @@
 {
   "parent": "GooeyDefence:PlainBlock",
-  "FireEffect": {
+  "FireEffector": {
     "drain": 5
   }
 }
```

Comment:
Newline at the end of file :)

Reply:
File is actually deleted in later branches so this is "fixed"